### PR TITLE
Move ios package and update to haul

### PR DESCRIPTION
**Pull request by tuncaulubilge**
Same update we did for android, applied to ios

### CODE REVIEW

**Review by qalvapps — APPROVED** [review]
LGTM

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
 node_modules/
 .vscode/
 *.log
-packages/android-app/assets/js/index.android.bundle.map
+android/assets/js/index.android.bundle.map
```
Comment (fourlastor) [inline .gitignore]: shouldn't this be done for iOS too?

File: android/package.json [hunk 2]
```diff
@@ -40,6 +40,6 @@
     "flow-bin": "0.63.1",
     "haul": "1.0.0-rc.4",
     "prettier": "1.8.2",
-    "webpack": "4.12.0"
+    "webpack": "4.6.0"
```
Comment (fourlastor) [inline android/package.json]: how come reverting this to 4.12?

**Review by fourlastor — CHANGES_REQUESTED** [review]

File: android/package.json (hunk 2 above)
Comment (tuncaulubilge) [reply android/package.json]: I updated this to 4.12.0 while testing android issues, but times-components are still on 4.6.0. I rather keep it consistent for now

File: .gitignore (hunk 1 above)
Comment (tuncaulubilge) [reply .gitignore]: Turns out these are ignored inside android and ios folders anyway, removing this one

**Review by fourlastor — APPROVED** [review]

**Pull request merged**